README.md

```diff
@@ -1,9 +1,12 @@
 # spark-etl
 
+![License](https://img.shields.io/github/license/mashape/apistatus.svg)](http://badges.mit-license.org)
```

Comment:
There is missing `[` at the beginning causing problems. Also can you move this badger to the end like make it the last one?